3.evaluate_model/scripts/nbconverted/F1_scores.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### Import Libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+from joblib import load
+
+import sys
+sys.path.append("../utils")
+from split_utils import get_features_data
+from train_utils import get_dataset
+from evaluate_utils import model_F1_score
+
+
+# ### Load Necessary Data
+
+# In[2]:
+
+
+# load features data from indexes and features dataframe
+data_split_path = pathlib.Path("../1.split_data/indexes/data_split_indexes.tsv")
+data_split_indexes = pd.read_csv(data_split_path, sep="\t", index_col=0)
+features_dataframe_path = pathlib.Path("../0.download_data/data/training_data.csv.gz")
+features_dataframe = get_features_data(features_dataframe_path)
+
+
+# ### Evaluate Each Model on Each Dataset
+
+# In[3]:
+
+
+# directory to load the models from
+models_dir = pathlib.Path("../2.train_model/models/")
+
+# use a list to keep track of scores in tidy long format for each model and dataset combination
```

Comment:
Is `tidy` a reference to a specific format or used as an adjective? If it's a specific format, consider linking to a reference for it where it makes most sense (if this hasn't already been done). Otherwise, please ignore this comment 🙂 !

Reply:
This is a specific format, thank you for reminding me to link a reference! Done in https://github.com/WayScience/phenotypic_profiling_model/pull/9/commits/ce807b04f97a7ca330cb5223f4180bf8b61e2dfa